vector/src/main/java/im/vector/app/features/home/room/list/home/HomeRoomListViewModel.kt

```diff
@@ -106,13 +106,8 @@ class HomeRoomListViewModel @AssistedInject constructor(
                 }
                 .onEach { selectedSpaceOption ->
                     val selectedSpace = selectedSpaceOption.orNull()
-                    val strategy = if (!vectorPreferences.prefSpacesShowAllRoomInHome()) {
```

Comment:
cc @fedrunov this condition looked inverted and has been updated to match the usages in other screens

```
when {
  vectorPreferences.prefSpacesShowAllRoomInHome() -> selectedSpaceId.toActiveSpaceOrNoFilter()
  else -> selectedSpaceId.toActiveSpaceOrOrphanRooms()
}
```